### PR TITLE
users' names render and combinned user components

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description
- Changed MUI to AntD components 
- names correctly render 
- forms prepopulate
- consolidated admin tools for managing users

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code

### CODE REVIEW

File: src/components/pages/AdminTools/ManageUsers.js [hunk 1]
```diff
@@ -1,47 +1,29 @@
 import React, { useEffect, useState } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
 import axios from 'axios';
-import Button from '@material-ui/core/Button';
 import { Link } from 'react-router-dom';
-import { Typography, Collapse } from 'antd';
+import { Typography, Collapse, Descriptions } from 'antd';
+import { Form, Input, Button as AntDButton, Modal, Select } from 'antd';
+import Icon from '@ant-design/icons';
```
Comment (katieolson84) [inline src/components/pages/AdminTools/ManageUsers.js:6]: Love seeing the line icon brought in to mimic the other pages. Great job!

File: src/components/pages/Home/RenderHomePage.js [hunk 2]
```diff
@@ -300,15 +296,6 @@ function RenderHomePage(props) {
             <Route exact path="/support">
               <SupportPage authState={user.authState} userInfo={hrfUserInfo} />
             </Route>
```
Comment (katieolson84) [inline src/components/pages/Home/RenderHomePage.js:298]: Great job cleaning up the routes!

**Review by katieolson84 — APPROVED** [review]
Looks great. All of the components are getting cleaned up and it is so satisfying to see consistency in the code now.

**Review by nwestallen — APPROVED** [review]
Looks good

**Pull request merged**